_contentTemplates/grid/built-in-dialogs.md

```diff
@@ -0,0 +1,13 @@
+#delete-confirmation
+The built-in Delete Confirmation Dialog requires minimum effort and code. Thus, the dialog itself does not provide customization options.
+
+To achieve custom appearance and behavior of the Delete Confirmation Dialog, use one of the following options:
+
+* **Localization** - this approach is useful if you want to change just the text of the built-in Delete Confirmation Dialog elements. It does not allow adding item details to the dialog text.
+
+* **Predefined Dialog Component** - this option is useful if you want to change the dialog text and include some details for the item the user tries to delete (for example, record name).
```

Comment:
We should not use bolding for emphasis. I don't like it, but this is what the style guide says.

Link to docs or example.

```suggestion
* Predefined dialogs (`DialogFactory`) - this option is useful if you want to change the dialog text and include some details for the item the user tries to delete (for example, record name).
```